### ISSUE TITLE
Warning: Each child in a list should have a unique "key" prop.

### ISSUE THREAD

**Issue by ssavitski** [plangrid/react-file-viewer#108]
https://github.com/plangrid/react-file-viewer/blob/a99b7e464b928abea66ececb7376e9758e1d5892/src/components/drivers/pdf-viewer.jsx#L129

**Comment by ssavitski** [comment]
add please `key={i.toString()}`

**Comment by jesuscmd** [comment]
Yes, please!!! This tool is amazing!!! Just add this to make it perfect!!

**Comment by SalmanHaider5** [comment]
Still issue is there, if anyone there to maintain it please do it first, its a really good package.

**Comment by melikeercan** [comment]
The issue still exists. Please fix, thank you. This is a great library

**Comment by SalmanHaider5** [comment]
I think there is no one now to maintain this library... From a long time, issue exist

**Comment by HasinaNjaratin** [comment]
The issue still exists. since 2019. is this lib maintained?

**Comment by muqeetmughal** [comment]
Is there anyone to maintaining this awesome liabrary ?
